ion/symbols_token.go

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package ion
+
+import "fmt"
+
+const (
+	// The placeholder for when a symbol token has no symbol ID.
+	SymbolIDUnknown = -1
+)
+
+// A location from a shared symbol table
```

Comment:
Suggestion: `ImportSource is a reference to a shared SymbolTable.`

Reply:
Fixed.